### PR TITLE
Adds rudimentary log event checks to the sims

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

We need an end-to-end test to check that logs are working fine (e.g. to flag breaking changes in PRs).

### What changes were made as part of this PR:

- Adds a check in the full-network sims that the HOC and POC deployment events are received correctly

### :rotating_light: Definition of done :rotating_light:
- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: integration/simulation/validate_chain.go [hunk 1]
```diff
@@ -363,3 +365,40 @@ func extractWithdrawals(t *testing.T, nodeClient rpc.Client, nodeAddr uint64) (t
...
+	for _, sub := range s.Subscriptions {
+		sub.Unsubscribe()
+	}
+
+	var gotHOCEvent bool
+	var gotPOCEvent bool
+	for {
+		select {
+		case receivedLog := <-s.LogChannel:
+			if receivedLog.Address.Hex() == "0x"+bridge.HOCAddr {
```
Comment (tudor-malene) [inline integration/simulation/validate_chain.go:386]: is it possible to check other fields of the log entry as well?

**Review by tudor-malene — APPROVED** [review]
lgtm.
one small q

File: integration/simulation/validate_chain.go (hunk 1 above)
Comment (joeldudley) [reply integration/simulation/validate_chain.go:386]: @tudor-malene Yeah, I can add that in a follow-up. What else would you like me to check?

Comment (tudor-malene) [reply integration/simulation/validate_chain.go:386]: like the topic. donno.

**Pull request merged**